x/ecocredit/core/params.go

```diff
@@ -213,18 +139,10 @@ func DefaultParams() Params {
 		sdk.NewCoins(sdk.NewCoin(sdk.DefaultBondDenom, DefaultBasketCreationFee)),
 		[]string{},
 		false,
-		[]*CreditType{
-			{
-				Name:         "carbon",
-				Abbreviation: "C",
-				Unit:         "metric ton CO2 equivalent",
-				Precision:    PRECISION,
-			},
-		},
 		[]*AskDenom{
 			{
-				Denom:        "uregen",
-				DisplayDenom: "regen",
+				Denom:        sdk.DefaultBondDenom,
```

Comment:
switched to default bond denom to match the above and make testing more straightforward.